CHANGELOG.md

```diff
@@ -49,6 +49,8 @@ Read more about the session argument for computed fields in the [docs](https://h
 ### Bug fixes and improvements
 
 (Add entries here in the order of: server, console, cli, docs, others)
+- console: display big int values in the console (close #3667, #3697, #4548)
```

Comment:
could you add this with the rest of the console entries here?

Reply:
Done!